kubernetes-examples/envelope-encryption/base/kroxylicious/kroxylicious-deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
     spec:
       containers:
       - name: kroxylicious
-        image: quay.io/kroxylicious/kroxylicious-developer:0.4.1
+        image: quay.io/kroxylicious/kroxylicious:0.5.0-SNAPSHOT
```

Comment:
hmm, we do want to get this back to a non snapshot version ASAP.

Reply:
I believe main should refer to main's snapshot.
the release mechanism updates the version number in these artefacts to main the release version.